docs/4.3/index.md

```diff
@@ -100,17 +97,6 @@ A good place to start if you want to jump right in.
 - [RBAC for SSH](enterprise/ssh_rbac.md) - Details on how Teleport Enterprise provides Role-based Access Controls (RBAC) for SSH.
 - [SSO for SSH](enterprise/sso/ssh_sso.md) - Overview on how Teleport Enterprise works with external identity providers for single sign-on (SSO).
 
-## Guides
-
-We also have several guides that go through the most typical configurations and integrations.
-
-- [Okta Integration](enterprise/sso/ssh_okta.md) - How to integrate Teleport Enterprise with Okta.
-- [ADFS Integration](enterprise/sso/ssh_adfs.md) - How to integrate Teleport Enterprise with Active Directory.
-- [One Login Integration](enterprise/sso/ssh_one_login.md) - How to integrate Teleport Enterprise with One Login.
-- [OIDC Integration](enterprise/sso/oidc.md) - How to integrate Teleport Enterprise with identity providers using OIDC/OAuth2.
-- [Kubernetes Integration](kubernetes_ssh.md) - How to configure Teleport to serve as a unified gateway for Kubernetes clusters and clusters of regular SSH nodes.
```

Comment:
Are these back in the side nav now?

Reply:
SSO is now in the side menu but the guides are a bit deeper. This was before the many guides we have today so it made sense to not give them as much prominence in the introduction.